.solhintignore

```diff
@@ -0,0 +1 @@
+contracts/ragnarok/types/**/*.sol
```

Comment:
Isn't it too much of a shortcut?

Reply:
solhint can't parse this type alias syntax and it's likely that all contracts in this folder will have one